### PR TITLE
Allow disabling git_ref by passing False

### PR DESCRIPTION
**Pull request by AleksanderWWW**
## Before submitting checklist

- [x] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [x] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

**Comment by AleksanderWWW** [comment]
Addressing https://github.com/neptune-ai/neptune-client/issues/1394

**Review by artsiomtserashkovich — APPROVED** [review]

**Comment by AleksanderWWW** [comment]
Just waiting for review from @normandy7 regarding docstrings

File: CHANGELOG.md
Comment (normandy7) [inline CHANGELOG.md:14]: Nitpick

```suggestion
- Allow disabling Git tracking by passing `git_ref=False` ([#1423](https://github.com/neptune-ai/neptune-client/pull/1423))
```

**Review by normandy7 — COMMENTED** [review]
Looks good otherwise

**Review by artsiomtserashkovich — APPROVED** [review]
LGTM

**Pull request merged**
